glide.lock

```diff
@@ -1,43 +1,102 @@
-hash: 86932a90d9bfe69d9f4a0bbf6e103c3b0670d47e8e1e2176fdd0b2637257d3a9
-updated: 2017-01-05T10:16:18.659229152-08:00
+hash: b50c74fff3d496405d1bc9dadfb044d56f1557317f22e34c186b8744af9acaa0
+updated: 2017-01-29T17:21:51.671174219Z
 imports:
 - name: github.com/aws/aws-sdk-go
-  version: 8649d278323ebf6bd20c9cd56ecb152b1c617375
+  version: 2ae1f45d01cdc7316a4d6c518c21a6d9d81a4970
   subpackages:
   - aws
+  - aws/defaults
+  - aws/session
   - aws/awserr
-  - aws/awsutil
-  - aws/client
-  - aws/client/metadata
-  - aws/corehandlers
+  - aws/service/dynamodb
+  - aws/service/s3
+  - service/dynamodb
+  - service/sts
+  - service/s3
   - aws/credentials
   - aws/credentials/ec2rolecreds
-  - aws/credentials/endpointcreds
-  - aws/credentials/stscreds
-  - aws/defaults
   - aws/ec2metadata
   - aws/endpoints
+  - aws/client/metadata
   - aws/request
-  - aws/service/dynamodb
-  - aws/service/s3
-  - aws/session
-  - aws/signer/v4
-  - private/protocol
-  - private/protocol/json/jsonutil
-  - private/protocol/jsonrpc
-  - private/protocol/query
-  - private/protocol/query/queryutil
   - private/protocol/rest
-  - private/protocol/restxml
-  - private/protocol/xml/xmlutil
-  - private/waiter
-  - service/dynamodb
+  - awsmigrate/awsmigrate-renamer/rename
+  - private/model/api
+  - private/util
   - service/kms
-  - service/kms/kmsiface
-  - service/s3
+  - service/s3/s3crypto
+  - service/acm
+  - service/apigateway
+  - service/applicationdiscoveryservice
+  - service/autoscaling
+  - service/cloudformation
+  - service/cloudfront
+  - service/cloudhsm
+  - service/cloudsearch
+  - service/cloudtrail
+  - service/cloudwatch
+  - service/cloudwatchlogs
+  - service/codecommit
+  - service/codedeploy
+  - service/codepipeline
+  - service/cognitoidentity
+  - service/cognitosync
+  - service/configservice
+  - service/datapipeline
+  - service/devicefarm
+  - service/directconnect
+  - service/directoryservice
+  - service/dynamodbstreams
+  - service/ec2
+  - service/ecs
+  - service/efs
+  - service/elasticache
+  - service/elasticbeanstalk
+  - service/elb
+  - service/elastictranscoder
+  - service/emr
+  - service/elasticsearchservice
+  - service/glacier
+  - service/iam
+  - service/iot
+  - service/iotdataplane
+  - service/kinesis
+  - service/lambda
+  - service/machinelearning
+  - service/opsworks
+  - service/rds
+  - service/redshift
+  - service/route53
+  - service/route53domains
+  - service/ses
+  - service/simpledb
+  - service/sns
+  - service/sqs
+  - service/ssm
+  - service/storagegateway
+  - service/support
+  - service/swf
+  - service/waf
+  - service/workspaces
+  - awstesting/unit
+  - service/cloudsearchdomain
+  - service/cloudwatchevents
+  - service/dynamodb/dynamodbattribute
+  - service/ecr
+  - service/firehose
+  - service/inspector
+  - service/marketplacecommerceanalytics
+  - service/mobileanalytics
+  - service/cloudfront/sign
+  - service/dynamodb/dynamodbiface
+  - service/sqs/sqsiface
```

Comment:
Do we really need this many additional AWS services? Will this bloat the binary?